graphql-server/src/databases/database.resolver.ts

```diff
@@ -91,6 +117,20 @@ export class DatabaseResolver {
     return dbs;
   }
 
+  @Mutation(_returns => Databases)
+  async databasesByConnection(
+    @Args() args: AddDatabaseConnectionArgs,
+  ): Promise<Databases> {
+    // Switch to the selected connection to get the databases
+    await this.addDatabaseConnection(args);
+    const conn = this.conn.connection();
+    const dbs = await conn.databases();
```

Comment:
Hm ok actually I was wrong about this. We shouldn't change the current connection to get the databases, because otherwise if I'm on the database page and I go to look at the menu and look at the databases in another connection but don't select one then the current page will be out of date.

There are two options I can currently see:
1. We initialize a new connection here, grab the databases, and then close the connection without touching the current connection
2. Instead of storing one current connection, we store a map of <connection_name, Connection>. We do this for hosted and it works well but it does add some complexity. We may want to do this eventually so that you can have different connections open in different tabs but it's a bigger refactor